lsp-mode.el

```diff
@@ -6368,7 +6376,8 @@ deserialization.")
                              (concat leftovers input)
                              err)))))))
         (mapc (lambda (msg)
```

Comment:
It seems to me that this mapc is no longer needed? The dispatch will effectively do the loop.

Reply:
So, it will work like this too, due to the guard. IMHO it might be more readable if you move the dispatch out of the loop.